designer/nodes/condition.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "Condition"
+icon: "code-merge"
+iconType: "solid"
+---
+
+The Condition Node  allows you to create different paths in a conversation based on specific conditions. 
+
+This means you can make your assistant respond differently depending on the situation.
+
+<img 
+height="500"  
+src="/images/node-condition.png"
+/>
+
+Think of conditions like a cause and effect: "**If** this happens, **then** do that."
+
+
+<Tip>
+Example:
+- **If** I wake up early, **then** I will go to the gym. 
+- **If** I am hungry, **then** I will eat.
+</Tip>
+
+## Adding Conditional Paths
+The Condition Node can have multiple condition paths. These paths let your assistant handle various scenarios. 
+
+To create a new path, click on the **+ Add Condition** option.
+
+<img 
+height="500"  
+src="/images/condition-add.png"
+/>
+
+
+### Why Use Conditional Paths?
+Using conditional paths makes your assistant more dynamic and responsive. 
+Instead of giving the same response every time, your assistant can provide tailored replies based on the user's input or other variables.
+
```

Comment:
Maybe in addition to responding differently, you could say you can route to an alternate path/flow in your conversation.

If user sent x messages, then ask for contact details.